tutorials/private-forseti-with-scc-integration/index.md

```diff
@@ -41,7 +40,7 @@ There is a [bug](https://github.com/forseti-security/forseti-security/issues/275
 
 In the `forseti` project:
 1. Navigate to the "VPC Networks" Pages on the Cloud Console
-2. Create a VPC for Forseti with one subnet. Configure it as follows:
+2. Create a VPC for Forseti with one subnet. Configure it with the following parameters:
    - Private access enabled
```

Comment:
while you are doing it can you please add period to all the lists that don't have it? is part of their guidelines (unless its a one word list should use period)